Makefile

```diff
@@ -28,7 +28,11 @@ HELM_BASE_URL = https://charts.crossplane.io
 HELM_S3_BUCKET = crossplane.charts
 HELM_CHARTS_DIR=$(ROOT_DIR)/charts
 HELM_CHARTS = oam-kubernetes-runtime
+LEGACY_HELM_CHARTS_DIR=$(ROOT_DIR)/legacy/charts
```

Comment:
I think it would be better to use the build submodule Helm chart machinery here; I believe it's possible for `HELM_CHARTS` to be a comma separated list of multiple Helm charts to build rather than introducing a new `make` target for the legacy chart.